### PR TITLE
[Issue dduk-ddak/coding-night-live#83] Docker Sandbox/Production Usage

### PR DESCRIPTION
**Pull request by minhoryang**

### CODE REVIEW

File: 2. Installation and Settings.md [hunk 1]
```diff
@@ -151,19 +151,58 @@ Using `Docker` makes deployment easier!
...
+`$ sudo docker-compose up -d daphne worker`
+
+### + Production way:
+
+Docker is also good for production.
+
+If you want to run it as production mode,
+
+1. should look at `docker-compose.yaml` [here](https://github.com/dduk-ddak/coding-night-live/blob/master/docker-compose.yaml) and find `for SANDBOX` keyword.
+    - *highly recommend to change it*
```
Comment (juice500ml) [inline 2. Installation and Settings.md:198]: --> Commenting out `SANDBOX` ports, currently `postgres`, `daphne`, `nginx`, and `redis`, is highly recommended, as internal ports should remain closed in production.
dduk-ddak/coding-night-live#94

Comment (minhoryang) [reply 2. Installation and Settings.md:198]: (not for nginx)

Comment (minhoryang) [reply 2. Installation and Settings.md:198]: @juice500ml Goto #3

Comment (juice500ml) [reply 2. Installation and Settings.md:198]: @minhoryang merged!

**Pull request merged**